rust-toolchain

```diff
@@ -1 +1 @@
-1.56.1
+1.62.1
```

Comment:
Do we have some kind of minimum version? Our toolchain files seem to be all over the place.